test/dotnet/input/bcpOptions.txt

```diff
@@ -276,4 +264,55 @@ bcp -w#!#in#!#bcp_source#!#destinationTable
 Select * from sourceTable
 Select * from destinationTable
 drop table sourceTable
-drop table destinationTable
\ No newline at end of file
+drop table destinationTable
+
+# -k keep_nulls
+Create table sourceTable(a int, b int)
+Create table destinationTable(a int, b int default(4))
+Insert into sourceTable values (1, 1);
+Insert into sourceTable values (1, NULL);
+Insert into sourceTable values (NULL, 1);
+Insert into sourceTable values (NULL, NULL);
+Insert into sourceTable(a) values (2);
+bcp -k#!#out#!#bcp_source#!#sourceTable
+bcp -k#!#in#!#bcp_source#!#destinationTable
```

Comment:
Lets also add test case for bcp without keep-nulls and for a default value-column